2765. Longest Alternating Subarray/solution.py

```diff
@@ -0,0 +1,26 @@
+class Solution:
+    # Takes in an array of numbers and returns the length of the longest
+    # alternating subarray, where an alternating subarray is a subarray
+    # with its elements alternating.
```

Comment:
This is not precise enough of a description. By this statement, 3, 6, 3, 6 would also be a alternating subarray as would 3, 2, 3. Both of these would not be considered an alternating subarray here